web-app/src/main/webapp/index.html

```diff
@@ -9,12 +9,21 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="container">
+
+      <form id="addressForm">
+        <div class="row">
+          <div class="two-thirds columns">
+            <input class="u-full-width" type="text" placeholder="Search"
+              id="searchInput">
+          </div>
+        </div>
+      </form>
+
+
+    </div>
+    <div id="map">
```

Comment:
nit: I would put the map first. Then I don't think you'll need to manually specify `z-order: 1`?